pyproject.toml

```diff
@@ -47,7 +47,7 @@ pubsub = [
 ]
 datastore = [
     "google-cloud-datastore>=2.20.1",
-    "pydantic>=2.10.3",
+    "pydantic<2",
```

Comment:
I'm not really sure this won't break the datastore, since we don't have an extensive suite of tests, it's hard to know.

Maybe we could downgrade just the healthcare dependency, keep the datastore, and prohibit from using both. What do you think?

In the long term, we can force the Pydantic 2 and fhir.resources 7, and use [fhir.resource's compatibility with R4](https://github.com/nazrulworld/fhir.resources/tree/main/fhir/resources/R4B). What do you think?